scripts/exregional_run_post.sh

```diff
@@ -385,8 +385,8 @@ bgdawp=${postprd_dir}/${NET}.t${cyc}z.bgdawpf${fhr}.${tmmark}.grib2
 bgrd3d=${postprd_dir}/${NET}.t${cyc}z.bgrd3df${fhr}.${tmmark}.grib2
 bgsfc=${postprd_dir}/${NET}.t${cyc}z.bgsfcf${fhr}.${tmmark}.grib2
 
-wgrib2 PRSLEV.GrbF${post_fhr} -set center 7 -grib ${bgdawp}
-wgrib2 NATLEV.GrbF${post_fhr} -set center 7 -grib ${bgrd3d}
+wgrib2 BGDAWP.GrbF${post_fhr} -set center 7 -grib ${bgdawp}
+wgrib2 BGRD3D.GrbF${post_fhr} -set center 7 -grib ${bgrd3d}
```

Comment:
We should not change to "BGDAWP" and "BGRD3D" as in the 000 subdirectory the final grib2 files are PRSLEV.GrbF00 and NATLEV.GrbF00

Reply:
After a few checks, I understood why you saw BGD* in your UPP runs.  

The postxconfig-NT-fv3lam_rrfs.txt file under regional_workflow/fix/upp (not the one under UPP/parm) is what is actually used in real time runs  and it defines "PRSLEV" and "NATLEV".